aspnetcore/signalr/hubs.md

```diff
@@ -509,46 +617,62 @@ If you have an exceptional condition you *do* want to propagate to the client, y
 > [!NOTE]
 > SignalR only sends the `Message` property of the exception to the client. The stack trace and other properties on the exception aren't available to the client.
 
+[!INCLUDE[](~/includes/signalr-in-shared-framework.md)]
```

Comment:
This doesn't look good where it is, it's directly after a section that's talking about getting errors on the client (and after another note). Maybe should be after the very first paragraph on this page?

Reply:
@BrennanConroy, Would the end of the first section, ### Configure SignalR hubs be OK with you?   That seems like an appropriate context.

Reply:
I'll move the note there to the end of the first section ### Configure SignalR hubs,  and if you still want it right after the intro (very first paragraph) instead, that is cool and I will change.